tests/ArrayFileTest.php

```diff
@@ -835,14 +835,13 @@ public function testSingleLineComment()
         );
     }
 
-    public function testSingleLineCommentSubItem()
+    public function testStrictFileTypes()
     {
-        $file = __DIR__ . '/fixtures/array/single-line-comments-subitem.php';
+        $file = __DIR__ . '/fixtures/array/strict-file-types.php';
         $arrayFile = ArrayFile::open($file);
 
-        $this->assertEquals(
-            str_replace("\r", '', file_get_contents($file)),
-            str_replace("\r", '', $arrayFile->render())
-        );
+        $code = $arrayFile->render();
+
+        $this->assertStringContainsString('declare', $code);
```

Comment:
Did you intend to remove the testSingleLineCommentSubItem() test or was this an accident?

Reply:
🤦🏻 Nope, I thought I copy/pasted this twice. I'll update that now